frontend/src/components/ui-kit/components/Button/Button.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Button.scss';
+
+const Button = ({ label, variant, icon, onClick }) => {
+  const buttonClass = ['button'];
+
+  if (variant === 'primary') {
+    buttonClass.push('primary');
+  } else if (variant === 'secondary') {
+    buttonClass.push('secondary');
+  }
+
```

Comment:
You can't do that at all, it will work every time.

You can also do this by simply substituting the "variant" prop directly into the className 
`const Button = ({ label, variant = 'primary', icon, onClick }) => {...
<button className={`button ${variant}`} onClick={onClick}>
...`